pages/linux/wtf.md

```diff
@@ -0,0 +1,12 @@
+# wtf
+
+> Show the expansions of acronyms.
+> More information: <https://manpages.debian.org/unstable/bsdgames/wtf.6.en.html>.
```

Comment:
Is it worth linking without the OS version so that it redirects to the latest stable version? 🤔
```suggestion
> More information: <https://manpages.debian.org/bsdgames/wtf.6.en.html>.
```

Reply:
I was thinking, from unstable manpages folks might have the latest reference (as compared to stable in Debian). So folks might miss knowing about certain latest features/updates which might land from upstream.

Though the argument works vice versa too, latest reference page, not so updated local package :D . So your suggestion might work best. Thanks!